libs/entry-form/styles/partials/core/components/forms/_general.scss

```diff
@@ -1,13 +1,26 @@
 .entry-form {
-    .action-buttons {
-        margin-top: 25px;
+	.action-buttons {
+		margin-top: 25px;
 
-        .button {
-            margin-left: 10px;
+		.button {
+			margin-left: 10px;
 
-            &:first-child {
-                margin-left: 0;
-            }
-        }
-    }
+			&:first-child {
```

Comment:
is it possible to change &:first-child with some class name (first-button, or first-item) as we have in the header, or that here is impossible?

Reply:
Yes, we can probably do that but we'll have ground refactoring ticket once we switch to angular v16 + code was untouched this time.